mu-plugins/blocks/multisite-latest-posts/src/block.js

```diff
@@ -53,13 +55,27 @@ const Block = ( { endpoint, perPage } ) => {
 		return <div>{ error }</div>;
 	}
 
+	let domParser;
+	if ( 'function' === typeof DOMParser ) {
+		domParser = new DOMParser();
+	}
+
 	return (
 		<ul className="wporg-multisite-latest-posts">
 			{ posts.map( ( i ) => {
+				let title;
 				const category = getCategory( i._embedded, i.categories[ 0 ] );
+
+				// Decode HTML entities that appear in titles in a way that is safe against XSS.
+				if ( 'object' === typeof domParser ) {
+					title = domParser.parseFromString( i.title.rendered, 'text/html' ).documentElement.textContent;
+				} else {
+					title = i.title.rendered;
+				}
```

Comment:
That's exactly the technique I was going to try. It seems hacky but comes widely recommended. If we used that one, is DOMParser beneficial?

Reply:
This textarea method is also what's used by [the `html-entities` GB package](https://github.com/WordPress/gutenberg/blob/trunk/packages/html-entities/src/index.js) — and actually, that has no dependencies so we could use it without adding a cascade of more stuff to the page.

Reply:
I didn't test, but that didn't seem safe based on https://stackoverflow.com/a/34064434/450127

IIRC in the past I've used `innerText` rather than `innerHTML`, but it sounds like [that's not entirely safe either](https://cheatsheetseries.owasp.org/cheatsheets/DOM_based_XSS_Prevention_Cheat_Sheet.html#usually-safe-methods).